package.json

```diff
@@ -19,7 +19,7 @@
     "find-business-days-in-range":"1.0.2"
   },
   "engines": {
-      "node": "v6.7.0",
+      "node": ">v6.9.0",
       "npm": "3.10.3"
```

Comment:
Yes. I also changed `>v6.9.0` to `>=6.9.0` to make you happy.

Staging output:

```
-----> Installing binaries
       engines.node (package.json):  >=v6.9.0
       engines.npm (package.json):   >=3.10.3

       Downloading and installing node 7.2.0...
       Downloaded [file:///tmp/buildpacks/dfa5716efe8295edfa82876f1d6fbe75/dependencies/https___buildpacks.cloudfoundry.org_dependencies_node_node-7.2.0-linux-x64.tgz]
       Resolving npm version >=3.10.3 via semver.io...
       Downloading and installing npm 4.1.1 (replacing version 3.10.9)...
```